dataactcore/scripts/initialize.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+import logging
+import os
+
+from flask_bcrypt import Bcrypt
+
+from dataactvalidator.app import createApp
+from dataactbroker.scripts.setupEmails import setupEmails
+from dataactcore.models.userModel import User
+from dataactcore.interfaces.function_bag import createUserWithPassword
+from dataactcore.scripts.setupAllDB import setupAllDB
+from dataactbroker.handlers.aws.session import SessionTable
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.config import CONFIG_BROKER, CONFIG_DB
+from dataactvalidator.scripts.loadTas import loadTas
+from dataactvalidator.filestreaming.sqlLoader import SQLLoader
+from dataactvalidator.filestreaming.schemaLoader import SchemaLoader
+from dataactvalidator.scripts.loadFile import loadDomainValues
+from dataactvalidator.scripts.loadSf133 import loadAllSf133
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+basePath = CONFIG_BROKER["path"]
+validator_config_path = os.path.join(basePath, "dataactvalidator", "config")
+
+
+def setupDB():
+    """Set up broker database and initialize data."""
+    logger.info('Setting up databases')
+    print('setting up db')
```

Comment:
Heh, the logger isn't consistently displaying so I had the print in there for debug. I'll ping you tomorrow for more discussion re: the logger. Thanks for pointing that out!